src/database.json

```diff
@@ -779,6 +779,12 @@
     "type": "github",
     "owner": "oakserver",
     "repo": "media_types",
+    "desc": "A simple and easy to use memoize lib for deno."
```

Comment:
It looks like the description has been added to the wrong place (swapped with the media types module)

Reply:
sure i will give it a check,

Reply:
done 

> It looks like the description has been added to the wrong place (swapped with the media types module)

done